### DISCUSSION TITLE
Typos on Angular getting started page: "Claity" & "coomponents"

### DISCUSSION THREAD

**Discussion by eemt** [vmware/clarity#5503] [General]
On the Angular development Getting Started page, near the bottom in the "What's Next" section
https://clarity.design/get-started/developing/angular/#what-s-next
There are a couple of typos.  

Step 2: "coomponents"
Step 3: "Claity" (the link to this page)

**Comment by Jinnie** [comment]
Thanks for reporting this. A fix was just committed.
